spec/unit/server_create_command_spec.rb

```diff
@@ -110,89 +108,13 @@ class ServerCreate < Chef::Knife::Cloud::ServerCreateCommand
     include Chef::Knife::Cloud::ServerCreateOptions
   end
 
-  describe "#bootstrap options" do
-
-    it "set chef config knife options" do
```

Comment:
why we removed this one ?

Reply:
As we have inherited **bootstrap_url, bootstrap_install_command, bootstrap_wget_options, bootstrap_curl_options,     bootstrap_no_proxy**  options from chef core and removed it from this commit changes https://github.com/chef/knife-cloud/pull/117/files#diff-3503536858f753f9066eca592720490eL185 . we need to removed this part of code.